com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -54,7 +54,7 @@ public Animator Animator
             }
         }
 
-        private bool m_SendMessagesAllowed = false;
+        protected virtual bool SendMessagesAllowed => IsServer;
```

Comment:
if caching is still necessary after more thorough testing, we can always cache IsServer in a second field and still have a virtual property for this.